docs/overview.md

```diff
@@ -0,0 +1,67 @@
+
+`rocd` is built on top of [Roc Toolkit](https://github.com/roc-streaming/roc-toolkit/), a library for real-time audio transport, offering features such as Forward Erasure Correction codes (FEC), adaptive latency tuning, clock drift compensation, lossy and lossless codecs.
+
+`rocd` incorporates Roc Toolkit for the transport part, ZeroMQ for the P2P signaling protocol, and implements high-level orchestration on top of that. Everything is wrapped into a simple **REST API** that you can use in your application.
+
+![](./assets/dia/overview.svg)
+
+## Features
+
+- **Isolates complexities of real-time streaming** and audio I/O are in a high-performance daemon with HTTP API.
+
+    Application developer can use any language (e.g. Node.js or Python) without any special requirements. Real-time path, sensitive to latency and performance, is fully enclosed inside the daemon.
+
+- **Encapsulated the hassle of P2P interactions.**
+
+    Application developer chooses the topology and orchestrates the system in a unified way, without worrying about low-level details like discovery, NAT traversal, distributed transactions, etc.
+
+- **Inherits [real-time features](https://roc-streaming.org/toolkit/docs/about_project/features.html)** provided by Roc Toolkit.
+
+    Roc Toolkit enables streaming of HD-quality audio with fixed or bounded latency and high robustness to packet loss. It is suitable for both low-latency, low-jitter networks like Ethernet and high-jitter unreliable networks like Wi-Fi and Internet.
+
+- Provides **REST API for managing audio devices and streams.**
+
+    Application developer can inspect and control audio devices (speakers, microphones) on all connected `rocd` instances, initiate streaming between them, control streams on fly, collect metrics, etc.
+
+- Provides **REST API for managing virtual audio devices.**
+
+    In addition to managing hardware devices, `rocd` is capable of creating and controlling virtual audio devices (VADs). For example, you can instruct it to create virtual speakers that stream all sound "played" to them to another audio device on another instance.
+    
+    Under the hood virtual devices are implemented as OS-specific sound server modules, for example [roc-pulse](https://github.com/roc-streaming/roc-pulse) for Linux and [roc-vad](https://github.com/roc-streaming/roc-vad) for macOS. `rocd` takes care of creating and configuring them using platform-specific interfaces.
+
+## Platforms
+
+!!! warning
+
+    Some platforms are not supported yet. See [roadmap](./roadmap.md) for status.
+
+`rocd` aims to support the following platforms:
+
+- **PC** — Personal computers with sound cards and apps doing audio I/O. Can be :white_check_mark: `GNU/Linux` or other `*nix`, :white_check_mark: `macOS`, or :construction: `Windows`.
```

Comment:
- why backticks here?
- you deleted the fact that PC has a display => has a different configuration (compared to server/embedded for example)